ui/src/groups/PrivacyNotice.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { useDismissNavigate } from '@/logic/routing';
+import Dialog from '@/components/Dialog';
+import { NavLink, useLocation } from 'react-router-dom';
+
+export default function PrivacyNotice() {
+  const dismiss = useDismissNavigate();
+  const { state } = useLocation();
+
+  const onOpenChange = (open: boolean) => {
+    if (!open) {
+      dismiss();
+    }
+  };
+
+  return (
+    <Dialog
+      defaultOpen
+      modal
+      onOpenChange={onOpenChange}
+      close="header"
+      className="h-[90vh] w-[90vw] overflow-hidden p-0 sm:h-[75vh] sm:max-h-[800px] sm:w-[75vw] sm:max-w-[800px]"
+      onInteractOutside={(e) => e.preventDefault()}
+    >
+      <div className="flex h-full w-full flex-col">
+        <div className="flex items-center space-x-2 border-b-2 border-b-gray-50 p-4">
+          <h2 className="font-semibold">Privacy Statement</h2>
+        </div>
+        <div className="h-full overflow-auto p-4">
+          <div className="prose mx-auto py-4 dark:prose-invert">
+            <h2>Dear friends, fans, and customers,</h2>
+            <p>
+              You&rsquo;re reading this note because you&rsquo;re concerned
+              about the privacy of your data. We&rsquo;ve published this notice
+              because we&rsquo;re concerned too (it&rsquo;s part of the reason
+              we built Urbit, after all).
+            </p>
+            <p>
+              Urbit was built to give people power over their computing.
+              Ownership and self-sovereignty remain foundational principles of
+              both the network-at-large and Tlon.
+            </p>
+            <p>
+              Still, we build software and offer Hosting services. Inevitably we
+              have to collect some data for you to use our product. As an
+              example, to sign up for a hosted ship, you give us your email.
+            </p>
+            <p>
+              If you use Landscape, you can help us make it better by allowing
+              surface level data collection about your use. What do we mean by
+              surface level?
+            </p>
+            <ul>
+              <li>The names of public groups you join.</li>
+              <li>
+                The names of public channels you post in and the type of
+                channel.
+              </li>
+              <li>
+                The <strong>types</strong> of channels you use in private
+                groups.
+              </li>
+            </ul>
+            <p>
+              By default,{' '}
+              <em>
+                we never track anything you do on Landscape as a self-hosted
+                user.
+              </em>{' '}
+              However, you can opt in to help us. If you&rsquo;re piloting a
+              Tlon-hosted ship, we collect surface level data about your ship
+              activity unless you opt out,{' '}
+              <NavLink
```

Comment:
Not sure why but this link doesn't work, brings you to a blank page

Reply:
This one works; the other didn't -- they look very similar; can you re-verify?

Reply:
hmm both seem to do this now 
<img width="828" alt="image" src="https://github.com/tloncorp/landscape-apps/assets/5466421/2576c51b-ccf1-4212-bf1e-b928c7b6ef76">